qiskit/algorithms/phase_estimators/phase_estimation_result.py

```diff
@@ -0,0 +1,147 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Result of running PhaseEstimation"""
+
+from typing import Dict, Union
+import numpy
+from qiskit.result import Result
+from .phase_estimator import PhaseEstimatorResult
+
+
+class PhaseEstimationResult(PhaseEstimatorResult):
+    """Store and manipulate results from running `PhaseEstimation`.
+
+    This class is instantiated by the `PhaseEstimation` class, not via user code.
+    The `PhaseEstimation` class generates a list of phases and corresponding weights. Upon
+    completion it returns the results as an instance of this class. The main method for
+    accessing the results is `filter_phases`.
+    """
+
+    def __init__(self, num_evaluation_qubits: int,
+                 circuit_result: Result,
```

Comment:
No. The phases are also included separately in `PhaseEstimationResult`, and there is the interface `filter_phases` as well. I include `circuit_result` because it includes information that may be useful in troubleshooting the algorithm and which would otherwise be lost.